From a58a1207420093044a7fe3e3cab0e9ac045b97d6 Mon Sep 17 00:00:00 2001
From: Aaron Plattner <aplattner@nvidia.com>
Date: Tue, 3 Sep 2019 11:17:43 -0700
Subject: [PATCH 2/3] doc: Add missing "HEVC Picture Parameter Set" group close
 marker

Fixing this imbalance doesn't cause any major changes to the generated
documentation.
---
 include/vdpau/vdpau.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/vdpau/vdpau.h b/include/vdpau/vdpau.h
index a56643e..3533bbe 100644
--- a/include/vdpau/vdpau.h
+++ b/include/vdpau/vdpau.h
@@ -3415,6 +3415,7 @@ typedef struct {
     uint8_t lists_modification_present_flag;
     uint8_t log2_parallel_merge_level_minus2;
     uint8_t slice_segment_header_extension_present_flag;
+    /** @} */
 
     /** \name HEVC Slice Segment Header
      *
-- 
2.11.0

